lib/osf-components/addon/components/file-browser/breadcrumbs/template.hbs

```diff
@@ -0,0 +1,25 @@
+<nav aria-label={{t 'registries.overview.files.breadcrumbs'}} local-class='Breadcrumbs'>
+    <ol data-test-breadcrumbs>
+        {{#each @manager.breadcrumbs as |breadcrumb index| }}
+            <li>
+                {{#if (eq index 0)}}
+                    <OsfLink
+                        data-test-go-to-folder={{breadcrumb.name}}
+                        @href='#'
+                        {{on 'click' (perform @manager.goToFolder breadcrumb)}}
+                    >
+                        {{t (concat 'registries.overview.files.storage_providers.' breadcrumb.name)}}
+                    </OsfLink>
+                {{else}}
+                    <OsfLink
+                        data-test-go-to-folder={{breadcrumb.name}}
+                        @href='#'
+                        {{on 'click' (perform @manager.goToFolder breadcrumb)}}
+                    >
+                        {{breadcrumb.name}}
+                    </OsfLink>
```

Comment:
I have one nitpick: should the current folder breadcrumb be clickable? Cuz it seems when clicking the current folder it will reset the filter and page and load the items again.